examples/TestCalendar.md

```diff
@@ -46,21 +46,23 @@ month:
 ```
 
 ### Colored by Values
-Use parameter `circleColorByValue`, color the circles based on the values
+Use parameters `circleColorByValue`, `yMin`, and `yMax`, to color the circles based on the values
```

Comment:
I think it's worth emphasizing that you need to set all 3 of these in order for it to take effect.